src/fonts/_base.scss

```diff
@@ -70,29 +70,3 @@ header {
 		content: "\f103";
 	}
 }
-
```

Comment:
I tested the look of the header brand on mobile and desktop with the latest version of GCweb and with the previous major version before that (v7.0.1 to be more precise), with zoom 300% and without, and no perceivable changes what so ever. The styles are in this file at line 31. The file can therefore be removed.